src/bin/inx-chronicle/api/stardust/poi/routes.rs

```diff
@@ -179,7 +179,7 @@ async fn create_proof_for_applied_blocks(
 
     // Fetch the referenced and applied block ids in "White Flag" order, and make sure they contain the block.
     let applied_block_ids = block_collection
-        .get_applied_blocks_in_white_flag_order(referenced_index)
```

Comment:
How is that not breaking PoI for applied transactions? :thinking:

Reply:
The problem is the `Included` check? I think I was assuming referenced blocks are never conflicting.

Reply:
Yeah...that's the problem. Milestones can reference conflicting transactions.

Reply:
Makes sense, my mistake. I'll revert this change.